### PR TITLE
BNB-131 | Fix : Add receive function

### PR DESCRIPTION
**Pull request by manoj9april**

### CODE REVIEW

File: contracts/StakeManager.sol [hunk 1]
```diff
@@ -54,6 +54,8 @@ contract StakeManager is
     uint256 public feeBps; // range {0-10_000}
     mapping(uint256 => bool) public rewardsIdUsed;
 
+    address public whitelistedAccount;
```
Comment (dulguun-staderlabs) [inline contracts/StakeManager.sol:57]: Maybe `whitelistedAddress` or `redirectAddress` or something that explains the logic might be better.

File: contracts/StakeManager.sol [hunk 2]
```diff
@@ -688,6 +703,16 @@ contract StakeManager is
         paused() ? _unpause() : _pause();
     }
 
+    receive() external payable {
+        address msgSender = msg.sender;
```
Comment (dulguun-staderlabs) [inline contracts/StakeManager.sol:707]: do we need it?

**Review by dulguun-staderlabs — CHANGES_REQUESTED** [review]
Looks good overall

File: contracts/StakeManager.sol (hunk 2 above)
Comment (manoj9april) [reply contracts/StakeManager.sol:707]: nopes, will remove

File: contracts/StakeManager.sol [hunk 3]
```diff
@@ -494,6 +496,19 @@ contract StakeManager is
         emit SetFeeBps(_feeBps);
     }
 
+    function whitelistAccount(address _address)
```
Comment (galacticminter) [inline contracts/StakeManager.sol:499]: `setWhitelilstAccount`?

File: contracts/StakeManager.sol (hunk 1 above)
Comment (galacticminter) [reply contracts/StakeManager.sol:57]: redirectAddress is better

Comment (manoj9april) [reply contracts/StakeManager.sol:57]: okay sure, will update

File: contracts/StakeManager.sol (hunk 3 above)
Comment (manoj9april) [reply contracts/StakeManager.sol:499]: `setRedirectAddress` then?

**Review by dulguun-staderlabs — APPROVED** [review]
Looks good 🚀

**Pull request merged**